docs/debugging.md

```diff
@@ -0,0 +1,192 @@
+
+```json title="Sample error from octez tezos-signer. HTTP code is 500"
+[{"kind":"permanent","id":"signer.ledger","ledger-error":"Application level error (sign-with-hash): Incorrect data"}]
+```
+
+TODO: Include an example error from signatory 
+
+To reset the watermark on a ledger, you can use the `tezos-client` cli.
+
+```bash title="Set the watermark to 0 in the Tezos Ledger baking app"
+tezos-signer set ledger high watermark for "ledger://stiff-sloth-grown-grouse/ed25519/0h/0h" to 0
+```
+You must review and accept a prompt on the ledger Tezos App.
+
+## `tezos-client` Display http requests & responses
+
+By adding the `--log-requests` or `-l`, to the `tezos-client`, you will see all the HTTP requests and responses between your `tezos-client` and the remote signer. Logging requests can be helpful for troubleshooting configuration problems.
+
+```
+$ tezos-client -l sign bytes 0x05deadbeef for remote_key
+>>>>0: http://localhost:8732/version
+<<<<0: 200 OK
+  { "version": { "major": 13, "minor": 0, "additional_info": "release" },
+    "network_version":
+      { "chain_name": "TEZOS_JAKARTANET_2022-04-27T15:00:00Z",
+        "distributed_db_version": 2, "p2p_version": 1 },
+    "commit_info":
+      { "commit_hash": "cb9f439e58c761e76ade589d1cdbd2abb737dc68",
+        "commit_date": "2022-05-05 11:55:24 +0200" } }
+>>>>1: http://localhost:8732/chains/main/blocks/head/protocols
+<<<<1: 200 OK
+  { "protocol": "PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY",
+    "next_protocol": "PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY" }
+>>>>2: http://localhost:6732/authorized_keys
+<<<<2: 200 OK
+  {}
+>>>>3: http://localhost:6732/keys/tz1g7VaasrnCUmSmCKUcP8o61vMPscVkoxYL
+  "05deadbeef"
+<<<<3: 200 OK
+  { "signature":
+      "edsigu5xkT8xcTEECLri3WZG2QuFLgH47NgFaRKQpBnaUwPw9DfhKHSNTAcvLxqVqWc4i7SEiUciHcEUzmeKYQDQioKcBzayUaY" }
+Signature: edsigu5xkT8xcTEECLri3WZG2QuFLgH47NgFaRKQpBnaUwPw9DfhKHSNTAcvLxqVqWc4i7SEiUciHcEUzmeKYQDQioKcBzayUaY
+$
+```
+
+## Octez tezos-signer
+
+The Octez tezos-signer is the signer of reference for Tezos. Signatory and tezos-signer implement the same core signing capabilities. Running `tezos-signer` for comparative purposes can be helpful when testing or developing signatory. 
+
+## Getting tezos-signer
+
+Download the latest `tezos-signer` binary from the _Releases_ page at https://github.com/serokell/tezos-packaging
+You will also want the `tezos-client` binary for testing & verification purposes.
+
+## Setup tezos-signer with a local secret
+
+To generate a new secret using `tezos-signer` run the command:
+
+```
+tezos-signer gen keys octez_signer_key
+```
+
+This will create (or add to) the configuration files in `~/.tezos-signer`
+
+:::tip
+You can use many of the familiar `tezos-client` sub-commands with the `tezos-signer` such as;
```

Comment:
Should this be a colon and not a semi-colon?

Reply:
fixed